tools/SourceKit/CMakeLists.txt

```diff
@@ -234,6 +238,11 @@ macro(add_sourcekit_executable name)
       set_target_properties(${name}
         PROPERTIES
         LINK_FLAGS "-Wl,-exported_symbol,_main")
+
+      if(SWIFT_ANALYZE_CODE_COVERAGE)
+        set_property(TARGET "${name}" APPEND_STRING PROPERTY
```

Comment:
I think we can remove the snippet in this one because `add_sourcekit_default_compiler_flags` is called after:
./tools/SourceKit/CMakeLists.txt

I'm mostly talking about `set_target_properties` in these since the override is the last thing that happens to the target:
./tools/SourceKit/tools/complete-test/CMakeLists.txt
./tools/SourceKit/tools/sourcekitd-repl/CMakeLists.txt
./tools/SourceKit/tools/sourcekitd-test/CMakeLists.txt
./unittests/CMakeLists.txt